src/ai/backend/web/config.py

```diff
@@ -174,6 +180,13 @@
             allow_devnull=True,
         ),
     }).allow_extra("*"),
+    t.Key("otel", default=_default_otel_config): t.Dict({
+        t.Key("enabled", default=_default_otel_config["enabled"]): t.ToBool,
+        t.Key("log-level", default=_default_otel_config["log-level"]): t.Enum(
+            "CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"
+        ),
```

Comment:
It seems better to define a new type rather than using the Enum type directly here.